src/Mod/Part/Gui/DlgPrimitives.cpp

```diff
@@ -64,6 +64,15 @@ using namespace PartGui;
 
 namespace PartGui {
 
+QString getAutoGroupCommandStr(QString& objectName)
```

Comment:
Hello @wwmayer! I have some doubts if it's good to put this function in this position, cause it could be reused in all the other Part commands (and if you agree i'd like to add the same behavior also for box, cylinder, etc...).